pypdf/_writer.py

```diff
@@ -132,6 +133,15 @@
 ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions((2**31 - 1) - 3)
 
 
+class ObjectDeletionFlag(enum.IntFlag):
+    TEXT = enum.auto()
+    IMAGES = enum.auto()
+    LINKS = enum.auto()
+    ATTACHMENTS = enum.auto()
+    OBJ_3D = enum.auto()
+    ALL_ANNOTS = enum.auto()
```

Comment:
I prefer to spell things out, especially when they are user facing:

```suggestion
    ALL_ANNOTATIONS = enum.auto()
```